truewiki/web_routes.py

```diff
@@ -9,6 +9,7 @@
 from . import (
     config,
     singleton,
+    wiki_page,
```

Comment:
Bit of nitpicking: I think I prefer `from .wiki_page import WikiPage`. In other files `wiki_page` is often used as instance of `WikiPage`, so this might be a bit confusing over time.

Reply:
I did it based on this suggestion earlier: https://github.com/TrueBrain/TrueWiki/pull/258#discussion_r787009647